fix(styles): Alerts: avoid flash of default icon when using post-icon (#2244)
swisspost/design-system: .changeset/plenty-pianos-check.md

---
'@swisspost/design-system-styles': patch
---

> Fixed flash of default icon when using post-icon on alerts.

Reviewer: ```suggestion
Fixed default icon flashing when using a `post-icon` on alerts.
```
